cola/models/stash.py

```diff
@@ -32,11 +32,14 @@ def is_changed(self):
 
     def stash_info(self, revids=False, names=False):
         """Parses "git stash list" and returns a list of stashes."""
-        stashes = self.stash_list()
-        revids = [s[:s.index(':')] for s in stashes]
-        names = [s.split(': ', 2)[-1] for s in stashes]
-
-        return stashes, revids, names
+        stashes = self.stash_list('--format=%gd/%aD/%s')
```

Comment:
Small sug: it might make sense to use a python `r'aw string'` here (prefix the string literal with `r`) so that it's clear that these are all literal values and we do not expect python to do any interpolation over the `%xxx` fields.  It doesn't make a difference in practice, but it makes it clear to the reader that we want to pass the string in exactly as-is.